### PR TITLE
[CAT-1544] Security input groovy test migration

### PR DESCRIPTION
**Pull request by a6software**

### CODE REVIEW

File: assets/js/preventDoubleSubmit/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+;(() => {
+  'use strict'
+
+  const disableSubmitButtons = event => {
+    console.log('event', event.currentTarget)
```
Comment (Stubbs) [inline assets/js/preventDoubleSubmit/index.js:5]: I'm surprised the linting check didn't say something about this, but looking at the config, `assets/` is on the ignore list.

Comment (a6software) [reply assets/js/preventDoubleSubmit/index.js]: whoops, removed

**Review by Stubbs — APPROVED** [review]
LGTM

**Comment by a6software** [comment]
ty

**Pull request merged**